.ci/linux/benchmark.sh

```diff
@@ -4,6 +4,6 @@
 #
 set -euxo pipefail
 
-cd ./test/Elastic.Apm.PerfTests
+cd ./test/Elastic.Apm.Perf.Tests
```

Comment:
Since we rename this: I'd vote for calling it `Elastic.Apm.Benchmarks`.

Reply:
Agreed